modules/ocf_prometheus/manifests/alertmanager.pp

```diff
@@ -26,21 +26,35 @@
 
     route         => {
       group_by => ['alertname'],
-      receiver => 'ocf_lowprio',
+      receiver => 'default',
       routes   => [
         {
           # Repeat MirrorOutOfDate alerts every 24h, instead of the default 4h
           match           => {'alertname' => 'MirrorOutOfDate'},
           repeat_interval => '24h',
+    receiver              => 'ocf_lowprio',
```

Comment:
Feel free to totally change the naming convention here, I'm not super attached to `ocf_lowprio`.